src/test/java/seedu/command/DeleteCommandTest.java

```diff
@@ -41,4 +41,33 @@ void execute_validSerialNumber_success() {
 
         assertEquals(expectedResult, actualResult);
     }
+
+    @Test
+    void execute_invalidSerialNumber_fail() {
+        deleteCommand = new DeleteCommand(new ArrayList<>(
+                Arrays.asList("s/S123445ASF")
+        ));
+        deleteCommand.setEquipmentManager(new EquipmentManager());
+
+        CommandResult actualResult = deleteCommand.execute();
+        CommandResult expectedResult =
+                new CommandResult("No such serial number, please enter an existing "
+                        + "serial number");
+
+        assertEquals(expectedResult, actualResult);
+    }
+
+    @Test
+    void execute_notSerialNumber_fail() {
+        deleteCommand = new DeleteCommand(new ArrayList<>(
+                Arrays.asList("n/Speaker3")
+        ));
+        deleteCommand.setEquipmentManager(new EquipmentManager());
+
+        CommandResult actualResult = deleteCommand.execute();
+        CommandResult expectedResult =
+                new CommandResult("Only serial number accepted for deleting equipment");
```

Comment:
Want to use the `ONLY_SN_ACCEPTED` string in `DeleteCommand` you previously defined?